docs/apache-airflow/configurations-ref.rst

```diff
@@ -48,6 +48,10 @@ that you run airflow components on is synchronized (for example using ntpd) othe
     [{{ section["name"] }}]
     {{ "=" * (section["name"]|length + 2) }}
 
+    {% if section.get('renamed') %}
```

Comment:
```suggestion
    {% if 'renamed' in section %}
```

Does this work? I feel Jinja2 is in some sort of an uncanny valley to Python that most things work but not all things…

Reply:
Both works. Confirmed by building the doc